src/helpers/experimental.js

```diff
@@ -46,13 +52,27 @@ export const experimentalFeatures: Feature[] = [
     description: 'Alternative USB implementation that might help solve USB issues.',
   },
   {
-    shadow: true,
     type: 'toggle',
+    name: 'LEDGER_COUNTERVALUES_API',
+    valueOn: 'http://countervalue-service.dev.aws.ledger.fr',
+    valueOff: 'https://countervalues.api.live.ledger.com',
+    title: 'Experimental Countervalues API',
+    description: 'Experimental Countervalues API upgrade',
+  },
```

Comment:
title: 'Experimental Countervalues API',
  description: 'Changing this value may break the countervalues displayed for your accounts.',